apps/test/code-studio/ProgressDotTest.js

```diff
@@ -48,7 +41,7 @@ describe('ProgressDot component tests', () => {
 
     const result = renderer.getRenderOutput();
     expect(result.props.onClick).to.not.be.a('function');
-    expect(radiumHelper(result.props.children[0].props.style).borderColor).to.equal(color.level_current);
+    expect(result.props.children[0].props.style.borderColor).to.equal(color.level_current);
```

Comment:
without context, perhaps breaking this up into an intermediate assignment or two might make it easier to understand?